src/helpers/devices.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { getDeviceModel } from '@ledgerhq/devices'
+
+import type { DeviceType } from 'reducers/onboarding'
+
+export const deviceModelName = (id: DeviceType): string => {
```

Comment:
I would suggest to directly do the `getDeviceModel` code in the UI side, no need for this function I feel (we can directly resolve the device model object on the UI side.